.gitignore

```diff
@@ -127,3 +127,6 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+
+# reports of jest tests
+reports
```

Comment:
enter

Reply:
NIIIIICE, it's .gitignore after all :)
Fixing